radixdlt/src/main/java/org/radix/GenerateUniverses.java

```diff
@@ -176,9 +174,8 @@ private Atom createGenesisAtom(byte magic, long timestamp, long planck) throws E
 		}
```

Comment:
Not new, but the client atom created on line 165 is never used, so could be removed.